### PR TITLE
Update page deletion popin

### PR DESCRIPTION
**Pull request by ndufreche**
see #848

### CODE REVIEW

**Comment by ndufreche** [comment]
screenshot from 2015-05-28 13 13 44

**Comment by ndufreche** [comment]
screenshot from 2015-05-28 13 14 10

File: src/tb/apps/page/views/page.view.delete.js [hunk 1]
```diff
@@ -33,72 +35,75 @@ define(
...
-                this.popin.setContent('Delete page ?');
-                this.popin.addButton('Delete', jQuery.proxy(this.onDelete, this));
-                this.popin.addButton('Cancel', jQuery.proxy(this.onCancel, this));
-
-                this.popin.display(this.popin);
-
-                return this;
-            }
-        });
+        var trans = Core.get('trans') || function (value) {return value; },
```
Comment (fkroockmann) [inline src/tb/apps/page/views/page.view.delete.js]: Use require('component!translator').translate()

**Comment by fkroockmann** [comment]
Thanks dude :) :+1:

**Pull request merged**